vignettes/programming_strategy.Rmd

```diff
@@ -683,50 +695,61 @@ parameters which are not passed as `exprs()` argument (e.g. `new_var = VAR1` or
 
 ## Unit Testing
 
-Unit tests for deprecated functions and parameters must be added to
-`tests/testthat/test-deprecation.R` to ensure that a warning or error is issued. 
+Unit tests for deprecated functions and arguments must be added to the test
+file of the function to ensure that a warning or error is issued.
```

Comment:
So the file `test-deprecation.R` will be removed at some point?

Can you provide a short example?  For example,  The function derive_var_example() is located in the `example.R` file.  The unit test will be found in `tests/testthat/test-example.R` file and should be updated there????

Reply:
Yes, in the Q3 release `test-deprecation.R` should be removed.

I have added a footnote regarding the naming of the test file.

Reply:
I don't think we need a separate issue for it. It should be covered by the general deprecation process issue for Q3.